packages/renderer/src/lib/onboarding/Onboarding.svelte

```diff
@@ -51,35 +50,19 @@ onMount(async () => {
   onboardingUnsubscribe = onboardingList.subscribe(onboardingItems => {
     if (!onboardings) {
       onboardings = onboardingItems.filter(o => extensionIds.find(extensionId => o.extension === extensionId));
-      if (!started) {
-        startOnboarding().then(isStartedByMe => {
-          if (isStartedByMe) {
-            assertStepCompleted().finally(() => (firstStart = false));
-          }
-        });
-      }
+      startOnboarding();
     }
   });
 
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
-  contextsUnsubscribe = context.subscribe(async value => {
+  contextsUnsubscribe = context.subscribe(value => {
     globalContext = value;
-    if (!started) {
-      const isStartedByMe = await startOnboarding();
-      if (isStartedByMe) {
-        firstStart = false;
-        await assertStepCompleted();
-        return;
-      }
-    }
-    if (!firstStart) {
-      await assertStepCompleted();
-    }
+    startOnboarding();
```

Comment:
I think we need to catch the error on a promise if we don't await for it.

Reply:
done